Description: Specifies the source encoding for the compiler to avoid unmappable
 character errors. For some reason the project.build.sourceEncoding property
 is ignored.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/pom.xml
+++ b/pom.xml
@@ -118,6 +118,7 @@
           <configuration>
             <source>1.8</source>
             <target>1.8</target>
+            <encoding>UTF-8</encoding>
           </configuration>
         </plugin>
         <plugin>
